src/tests/setup.js

```diff
@@ -1,5 +0,0 @@
-/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
@XROLE Why was this file deleted?

Reply:
looks like there was a mistake some where. Adding it back right away